### ISSUE TITLE
readme.md - Creating an App Stack Project - do not use dashes in project name

### ISSUE THREAD

**Issue by mpskeeter** [JaimeStill/PatternsAndPractices#35]
# [Creating an App Stack Project](#patterns-and-practices)  

From a command prompt resolved to the created directory, execute the following:

```
dotnet new stack -n {project-name} -o {project-name}
```  

Should be noted that when creating a project, exclude the usage of a -(dash) in the project name.

This will prevent issues when executing {Project}dotnet build

```
C:\Apps>dotnet new stack -n test-app -o test-app
The template "Stack" was created successfully.

C:\Apps>cd test-app

C:\Apps\test-app>dotnet build
Microsoft (R) Build Engine version 16.0.450+ga8dc7f1d34 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Data\test_app.Data.csproj" was not found. [C:\Apps\test-app\test-app.sln]
C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Web\test_app.Web.csproj" was not found. [C:\Apps\test-app\test-app.sln]
C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Core\test_app.Core.csproj" was not found. [C:\Apps\test-app\test-app.sln]
C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Identity\test_app.Identity.csproj" was not found. [C:\Apps\test-app\test-app.sln]
C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Identity.Mock\test_app.Identity.Mock.csproj" was not found. [C:\Apps\test-app\test-app.sln]

Build FAILED.

C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Data\test_app.Data.csproj" was not found. [C:\Apps\test-app\test-app.sln]
C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Web\test_app.Web.csproj" was not found. [C:\Apps\test-app\test-app.sln]
C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Core\test_app.Core.csproj" was not found. [C:\Apps\test-app\test-app.sln]
C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Identity\test_app.Identity.csproj" was not found. [C:\Apps\test-app\test-app.sln]
C:\Program Files\dotnet\sdk\2.2.202\NuGet.targets(246,5): error MSB3202: The project file "C:\Apps\test-app\test_app.Identity.Mock\test_app.Identity.Mock.csproj" was not found. [C:\Apps\test-app\test-app.sln]
    0 Warning(s)
    5 Error(s)

Time Elapsed 00:00:00.50
```

**Comment by JaimeStill** [comment]
I'll add a note about not using `kebab-casing` for the naming just above the command prompt execution section.

**Comment by JaimeStill** [comment]
resolved with 3360c1c8c14bcec1274c2f48c4f768641554ad23
